languages/common-lisp/exercises/concept/sexpr/sexpr-test.lisp

```diff
@@ -33,3 +38,8 @@
   (is-false (sexpr:is-a-cons-p #'atom))
   (is-false (sexpr:is-a-cons-p #'consp))
   (is-false (sexpr:is-a-cons-p nil)))
+
+(defun run-test-suite ()
+  (progn
+    (load "sexpr.lisp")
```

Comment:
I was thinking of giving this as part of the instructions. Remember that the student may not in fact be working in a text file but in the REPL. Loading the file may actually be detrimental to a REPL based workflow.

Reply:
I suppose that's true. I've been on the fence about including that but decided to because I figured it would be a bit tedious to run two commands in the REPL every time they want to retest a solution.

I don't love having the `load` here, but I think it makes it easy to test incremental changes.

I'm certainly open to discussing it more, however.

Reply:
I am on the fence. I'll take a look how the other lisps do it as they may have the same sort of issues.